tket/src/Predicates/PassGenerators.cpp

```diff
@@ -931,8 +931,8 @@ PassPtr FullPeepholeOptimise(bool allow_swaps, OpType target_2qb_gate) {
 
 PassPtr gen_optimise_phase_gadgets(CXConfigType cx_config) {
   Transform t = Transforms::optimise_via_PhaseGadget(cx_config);
-  PredicatePtr ccontrol_pred = std::make_shared<NoClassicalControlPredicate>();
```

Comment:
Think ccontrol was intended here "classical control"

Reply:
Yes, this is not a typo.